README.md

```diff
@@ -72,8 +72,6 @@ docker run -i --rm -p 8080:8080 kind-registry.local:5000/quarkus-hello:1.0
 
 To validate this scenario we will use the [pack client](https://buildpacks.io/docs/tools/pack/).
 
->>**NOTE**: The pack client uses by default the Paketo builder `tiny` [image](https://github.com/paketo-buildpacks/tiny-builder). So it is then not needed to specify it as parameter !
```

Comment:
@cmoulliard as far as I see, this is not true. You need to configure the pack client using `pack client builder xxx`, otherwise the pack build will fail.

Reply:
This is correct. The builder image was cached on my machine. I will update the README page